src/components/Person/Person.jsx

```diff
@@ -1 +1,21 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <>
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${person.name}`}
+      </h2>
+      {person.age && (
+        <p className="Person__age">
+          {`I am ${person.age}`}
+        </p>
+      )}
+      <p className="Person__partner">
+        {person.sex === 'm' && person.isMarried === true ? `${person.partnerName} is my wife` : ''}
+
+        {person.sex === 'f' && person.isMarried === true ? `${person.partnerName} is my husband` : ''}
+
+        {person.isMarried === false ? `I am not married` : ''}
+      </p>
```

Comment:
you can move some part of this logic to var, it will improve the readability of the code